sections/03_categorize.md

```diff
@@ -1,23 +1,43 @@
-
-*Focus for the purpose of this review - within a health care context.*
-
-We currently categorize individuals using relatively *ad hoc* categories. These
-are divided, to an extent, by organ (e.g. cancers by tumor site), perhaps to an
-extent symptom, and to an extent by immediate cause. This system undergoes
-continual refinement (e.g. new subtypes of disease) as our understanding
-improves.
-
-*Would deep learning enable us to do this automatically in some principled way?
-Are there reasons to believe that this would be advantageous? Would it be
-positive to have disease categories changed by data, or would the changing
-definition (i.e. as more data are accumulated) actually be harmful? What
-impacts would this have on the training of physicians?*
-
-*What are the major challenges in this space, and does deep learning enable us
-to tackle any of them? Are there example approaches whereby deep learning is
-already having a transformational impact? I (Casey) have added some sections
-below where I think we could contribute to the field with our discussion.*
+## Deep learning and patient categorization
+
+In a health care setting, individuals are diagnosed with a disease or condition
+based on symptoms, the results of certain diagnostic tests, or other factors.
+Once diagnosed with a disease an individual might be assigned a stage based on
+another set of human-defined rules. While these rules are refined over time, the
+process is evolutionary rather than revolutionary.
+
+We might imagine that deep learning or artificial intelligence methods could
+reinvent how individuals are categorized for health care. A deep neural network
+might identify entirely new categories of health or disease that are only
+present when data from multiple lab tests are integrated. As a potential
+example, consider the condition Latent Autoimmune Diabetes in Adults (LADA). The
+history of this disease classification is briefly reviewed in Stenström et
+al.[@doi:10.2337/diabetes.54.suppl_2.S68].
+
+Imagine that a deep neural network operating in the early 1980s had access to
+electronic health records with comprehensive clinical tests. It might have
+identified a subgroup of individuals with blood glucose levels that indicated
+diabetes as well as autoantibodies, even though the individuals had never been
+diagnosed with type 1 diabetes - the autoimmune form of the disease that arises
+in young people. Such a neural network would be identifying patients with LADA.
+As no such computational approach existed, LADA was actually identified by Groop
+et al. [@doi:10.2337/diab.35.2.237]. However, this represents a potential hope
+of this area. Perhaps deep neural networks, by reevaluating data without the
+context of our assumptions, can reveal novel classes of treatable conditions.
+
+Alternatively, imagine that a deep neural network is provided with clinical test
```

Comment:
I like this but am not sure it's unique to deep learning. We do something like this with a random forest in the Geisinger missing data project (can cite if we get a preprint)